### ISSUE TITLE
Remember scroll position in feed

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/social#360]
**Describe the bug**
The scroll position in the feed is not remembered on navigation, resulting in the annoying need to scroll down again to get back to where you were.

**To Reproduce**
1. Go to your home timeline or any other timeline
2. Scroll down a bunch, reading posts
3. Click on the username of one of the posts authors
4. Go back

You will end up at the top of the feed again.

**Expected behavior**
You are back at the position in the feed where you left off.

**Comment by daita** [comment]
Also an icon to go back to the top of the timeline

**Comment by violoncelloCH** [comment]
I'd say a sticky button like in latest talk beta...

**Comment by jancborchardt** [comment]
Yes, but the important thing is to first remember the scroll position. ;) You could also always go back up via just using the navigation entry.

**Comment by SchBenedikt** [comment]
Automatically closed — this is a feature request (remember scroll position), not a bug. The front-end has been rewritten since this was filed.
